web-apis/geolocation/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * A function to check if the client supports/allows geolocation API support
+ * @returns boolean True if the client allows, else false
+ */
+const isSupported = () => (navigator.geolocation ? true : false);
+
+const GEOLOCATION_OPTIONS = {
+  // indicates the application would like to receive the best possible results
+  enableHighAccuracy: true,
+
+  // amount of time before the error callback is invoked, if 0 it will never invoke.
+  timeout: 5000,
+
+  // maximum cached position age. If set to 0, it means that the device cannot use a cached position and must attempt to retrieve the real current position.
+  maximumAge: 0,
+};
+
+/**
+ * A function that get's the coordinates of your current geo location
+ * and prints on the UI. If Geolocation API is not supported/allowed
+ * this function shows a valid error message.
+ */
+const geoFindMe = () => {
+  const status = document.querySelector('#error-text'),
+    mapLink = document.querySelector('#map-link'),
+    mapAccuracy = document.querySelector('#map-accuracy');
+
+  mapLink.href = '';
+  mapLink.textContent = '';
+  mapAccuracy.textContent = '';
+
+  const success = position => {
+      const latitude = position.coords.latitude,
+        longitude = position.coords.longitude,
+        accuracy = position.coords.accuracy;
+
+      status.textContent = '';
+      mapLink.href = `https://www.openstreetmap.org/#map=18/${latitude}/${longitude}`;
+      mapLink.textContent = `Latitude: ${latitude} °, Longitude: ${longitude} °`;
+      mapLink.textContent = `Latitude: ${latitude} °, Longitude: ${longitude} °`;
+      mapAccuracy.textContent = `Your location is accurate upto ${accuracy} meters`;
```

Comment:
@wsameer , you might have to convert the number :)

![image](https://user-images.githubusercontent.com/3633137/135579905-4053e8a7-8d40-44ab-b779-d20edba3f814.png)

Reply:
@atapas convert? you mean make it a whole number? or anything else?

Reply:
At least,
- 2 decimals after a point.
- It would be great to short the number like the above number is 2.46kms

This may help: https://www.npmjs.com/package/short-number

Reply:
@atapas  okay, so you want to convert into Kilometers. got it!